src/ethereum_test_fixtures/blockchain.py

```diff
@@ -407,6 +407,7 @@ class BlockchainFixtureCommon(BaseFixture):
     genesis: FixtureHeader = Field(..., alias="genesisBlockHeader")
     pre: Alloc
     post_state: Alloc | None = Field(None)
+    post_state_hash: Hash | None = Field(None)
```

Comment:
Maybe a paranoid comment, but worth doing an assertion that if both fields are explicitely set, then check their coherency?

Reply:
Thanks! Done 👍